packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -236,6 +236,7 @@ export class Summarizer extends EventEmitter implements ISummarizer {
             eventName: "RunningSummarizer",
             onBehalfOf,
             initSummarySeqNumber: this.runtime.deltaManager.initialSequenceNumber,
+            config: JSON.stringify(this.configurationGetter()),
```

Comment:
My only concern here - can it throw? For example, it would throw if config has circular reference.
Not the case today, and likely it's so catastrophic that we would capture it before check in (as it would always fail), so probably Ok?

Reply:
Yes, it could throw if there's a circular reference. But we shouldn't be having a circular reference in a configuration object and having such would indicate a design problem in my opinion.